src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -280,7 +280,13 @@
       <IncludeInVSIX>true</IncludeInVSIX>
       <Link>THIRD-PARTY-NOTICES</Link>
     </Content>
+  </ItemGroup>
 
+  <ItemGroup Label="Sloop Dependencies">
+    <Content Include="$(MSBuildThisFileDirectory)$(JarsFolderName)\*.jar">
+      <IncludeInVSIX>True</IncludeInVSIX>
+      <VSIXSubPath>$(JarsFolderName)</VSIXSubPath>
```

Comment:
Is this somehow related? https://github.com/SonarSource/sonarlint-visualstudio/blob/3400f8af51d6f9b44cdf34196ae5ecbe57423ba2/src/Integration.Vsix/SonarLintIntegrationPackage.cs#L47

Reply:
I don't see how these can be related. The one you referred is about loading assemblies, we are not interested in that here.